en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -1058,27 +1098,26 @@ SilverStripe\GraphQL\Manager:
 ```
 
 Once configured, flush your cache and explore the new GraphQL schema to ensure it loads correctly. You can use a GraphQL
-application such as GraphiQL, or [silverstripe-graphql-devtools](https://github.com/silverstripe/silverstripe-graphql-devtools)
+application such as GraphiQL, or [`silverstripe/graphql-devtools`](https://github.com/silverstripe/silverstripe-graphql-devtools)
 to view the schema and run queries from your browser:
 
 ```bash
 composer require --dev silverstripe/graphql-devtools dev-master
 ```
 
-#### Use the GraphQL query and mutation in javascript
+#### Use the GraphQL query and mutation in JavaScript
 
 The history viewer interface uses two main operations:
 
-* Read a list of versions for a DataObject
-* Revert (aka rollback) to an older version of a DataObject
+- Read a list of versions for a DataObject
+- Revert (aka rollback) to an older version of a DataObject
 
-`silverstripe/versioned` provides some graphql plugins we're taking advantage of here. See [Working with DataObjects - Versioned content](/developer_guides/graphql/working_with_dataobjects/versioning/) for more information.
+`silverstripe/versioned` provides some GraphQL plugins we're taking advantage of here. See [Working with DataObjects - Versioned content](/developer_guides/graphql/working_with_dataobjects/versioning/) for more information.
 
 For this we need one query and one mutation:
 
-**app/client/src/state/readOneMyVersionedObjectQuery.js**
-
 ```js
+// app/client/src/state/readOneMyVersionedObjectQuery.js
```

Comment:
> I'd much prefer to have the filename above the code fence in bold is best?

That violates [MD036 - emphasis used instead of heading](https://github.com/DavidAnson/markdownlint/blob/main/doc/md036.md), which is why I converted those to be comments inside the codeblock itself.

There wasn't actually a documented standard for those anyway, so we're not violating our documentation standards by changing this - and one of the commits in this PR _does_ explicitly add a new standard to say that these should be added as comments in the code block going forward.

I think having to remove a single line in an example that they copy/paste is not a burden that people will realistically have problems with - and the JSON thing is also fine since these are documentation examples and not real JSON in real project code. People will understand the point, I think.

Reply:
I just don't like that standard

Whenever I do code examples I've always used the bold heading above the code fence. I think it's better for several reasons:
- It leads to better copy paste (no redundant comment above the filename that needs to be deleted, yes it's trivial though not having it there is objectively better)
- We don't have the issue of .json files which don't have native comments
- It's consistent - we use the same style for every file type.  This reduces cognitive friction which is the entire point for standardisation.  Remember people are going to add these manually so it's going to be annoying to be told by the linter that you're formatting is wrong for the silverstripe template code example you're adding - so then you need to lookup how to do a comments in silverstripe template which you've forgotten how. Much easier if it's just "do bold every time"

Reply:
The first point is trivial, so probably not worth arguing about.
The second point I've already spoken to and don't think it's a problem.
The third point is actually incorrect - it wasn't being consistently done. But even if it was, changing one consistent thing for another consistent thing invalidates this point. Unless your point regarding consistency is that different language types have different comment styles? But that seems like a moot point since we have comments in the examples anyway, and we're not going to _not_ have comments in them.

Options:
1. Leave it the way it now is. I think this is sensible.
    - It's the same style Symfony uses if anyone cares about that
    - it keeps the filename inside the code block which removes any chance someone will add extra content between the filename and the code block
    - it passes linting without having to either globally remove this rule (and therefore potentially reduce the quality of our markdown) or add an exclude comment for every filename.
2. Use emphasis above the code block and add an explicit rule exemption for every filename
    - Likely to be forgotten when initially adding new code examples, and thus adding a little bit of friction when doing so
3. Use emphasis above the code block and disable the `MD036` rule globally
    - Results in allowing emphasis when headings should be used, which decreases readability and accessibility.
    
@maxime-rainville Do you have any preference here?

Reply:
>  Unless your point regarding consistency is that different language types have different comment styles? But that seems like a moot point since we have comments in the examples anyway

It's not a moot point. If I'm writing a quick PR to add a code example to a changelog then it's simply easier if there's a single way to write filenames not matter what the extensions is.  Means I don't need to have to search to work out how to write it for a template file or for a json file. Not searching is easier than having to search?

Reply:
This seems like a very minor point to me. 


I would be incline to pick "Door 1 - Leave it the way it now is" in the name of avoiding linting exemption.